gensim/models/ldamodel.py

```diff
@@ -369,7 +369,7 @@ def __init__(self, corpus=None, num_topics=100, id2word=None,
             Alternatively default prior selecting strategies can be employed by supplying a string:
 
                 * 'asymmetric': Uses a fixed normalized asymmetric prior of `1.0 / topicno`.
-                * 'default': Learns an asymmetric prior from the corpus.
+                * 'auto': Learns an asymmetric prior from the corpus.
```

Comment:
please update this in other places too

https://github.com/RaRe-Technologies/gensim/blob/56070fbb772259d434af957f46b4cda8954f4927/gensim/models/ldamodel.py#L534
https://github.com/RaRe-Technologies/gensim/blob/56070fbb772259d434af957f46b4cda8954f4927/gensim/models/ldamulticore.py#L132
https://github.com/RaRe-Technologies/gensim/blob/466b32fe6b5631403fe5be8c16233df3d61a373b/gensim/sklearn_api/ldamodel.py#L64
https://github.com/RaRe-Technologies/gensim/blob/466b32fe6b5631403fe5be8c16233df3d61a373b/gensim/sklearn_api/atmodel.py#L86